xml/System.Runtime.Serialization/ISerializationSurrogateProvider2.xml

```diff
@@ -15,8 +15,8 @@
     </Interface>
   </Interfaces>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides the methods needed to substitute one type for another by DataContractSerializer during export and import of XML schema documents (XSD). This interface builds upon <see cref="T:System.Runtime.Serialization.ISerializationSurrogateProvider" />.</summary>
+    <remarks>Together with <c>ISerializationCodeDomSurrogateProvider</c>, these interfaces replace the <c>IDataContractSurrogate</c> from .NET Framework 4.8.</remarks>
```

Comment:
"these interfaces replace <c>IDataContractSurrogate</c> from"
or
"these interfaces replace the <c>IDataContractSurrogate</c> interface from"

Reply:
GitHub suggestions are welcome. Which one do you prefer?

Reply:
No preference really. Either of them flow more naturally than the original. But for easy button clicking...
```suggestion
    <remarks>Together with <c>ISerializationCodeDomSurrogateProvider</c>, these interfaces replace the <c>IDataContractSurrogate</c> interface from .NET Framework 4.8.</remarks>
```